wiki/en/en-Installation-for-Windows.md

```diff
@@ -25,9 +25,9 @@ Jamulus uses [ASIO](https://en.wikipedia.org/wiki/Audio_Stream_Input/Output) to
 
 You can try two versions of ASIO4ALL. ASIO4ALL v2.14 includes a workaround for a bug which might break other functionality.
```

Comment:
This should probably be updated then?

Reply:
This is still true, so I'd remove it once 2.15 is out of beta.